src/index.ts

```diff
@@ -113,6 +113,7 @@ import { pProps } from './promise/pProps'
 import { pRetry, PRetryOptions } from './promise/pRetry'
 import { pState } from './promise/pState'
 import { pTimeout, PTimeoutOptions } from './promise/pTimeout'
+import {pTo} from './promise/pTo'
```

Comment:
Seems like husky/lint-staged didn't work properly and didn't run the formatting. Next time I'd run `yarn lint-all` manually to make sure all is formatted, or check what's wrong with husky (maybe `yarn --force` or smth can fix it).

In the future maybe I should set up CI action to run it (as we have in non-open-source projects) to catch such cases. Didn't have the need so far.

But no worries here, just a note to myself mostly.